pyproject.toml

```diff
@@ -65,7 +65,8 @@ dependencies = [
     "nwa-stdlib~=1.4.8",
     "oauth2-lib==1.3.2",
     "tabulate==0.9.0",
-    "strawberry-graphql==0.171.1"
+    "strawberry-graphql==0.171.1",
+    "pydantic-forms @ git+https://github.com/workfloworchestrator/pydantic-forms.git@0.2.0",
```

Comment:
Pydantic forms is [published on pypi ](https://pypi.org/project/pydantic-forms/#description) so `pydantic-forms==0.2.0` should work :)

Reply:
Ah yep! Changed